util/upload-instance-slot-map.py

```diff
@@ -43,9 +65,25 @@ def get_all_aws_regions(region):
     return sorted(r.get("RegionName") for r in ec2.describe_regions().get("Regions"))
 
 
+def get_old_instances_info(region, aws_credentials=None):
+    bucket_name = region + "-aws-parallelcluster"
+    print(bucket_name)
+    s3 = return_s3_object(region, aws_credentials, bucket_name)
+    old_instances_file_content = s3.Object(bucket_name, "instances/instances.json").get()["Body"].read()
+    return json.loads(old_instances_file_content)
+
+
 def push_to_s3(region, aws_credentials=None):
```

Comment:
rename function... doesn't look like it pushes to S3

Reply:
Does push to s3 with `bucket.upload_file()`